lib/mongoose_push/metrics/telemetry_metrics.ex

```diff
@@ -15,17 +15,28 @@ defmodule MongoosePush.Metrics.TelemetryMetrics do
         event_name: [:mongoose_push, :notification, :send],
         measurement: :time,
         buckets: [1000, 10_000, 25_000, 50_000, 100_000, 250_000, 500_000, 1000_000],
-        tags: [:status, :service, :error_category, :error_reason]
+        tags: [:status, :service, :error_category, :error_reason],
+        description:
+          "A histogram showing push notification sent times. Includes worker selection (with possible waiting if all are busy)"
       ),
 
       # measurement is ignored in Counter metric
-      Telemetry.Metrics.counter("mongoose_push.supervisor.init.count", tags: [:service]),
-      Telemetry.Metrics.counter("mongoose_push.apns.state.init.count"),
+      Telemetry.Metrics.counter("mongoose_push.supervisor.init.count",
+        tags: [:service],
+        description: "Counts number of push notification service supervisor starts"
+      ),
+      Telemetry.Metrics.counter("mongoose_push.apns.state.init.count",
+        description: "Counts number of APNS state initialisations"
+      ),
       Telemetry.Metrics.counter("mongoose_push.apns.state.terminate.count",
         tags: [:error_reason],
-        tag_values: fn metadata -> %{metadata | error_reason: metadata.reason} end
+        tag_values: fn metadata -> %{metadata | error_reason: metadata.reason} end,
+        description: "Counts number of APNS state terminations"
       ),
-      Telemetry.Metrics.counter("mongoose_push.apns.state.get_default_topic.count")
+      Telemetry.Metrics.counter(
+        "mongoose_push.apns.state.get_default_topic.count",
```

Comment:
```suggestion
      Telemetry.Metrics.counter("mongoose_push.apns.state.get_default_topic.count",
```

just for consistency with the remaining events.